packages/component-base/src/polylit-mixin.js

```diff
@@ -103,6 +103,22 @@ const PolylitMixinImplementation = (superclass) => {
         });
       }
 
+      if (options.sync) {
```

Comment:
When all the properties are synchronous, will it force the component to render synchronously during initialization?